code-challenges.test.js

```diff
@@ -14,33 +14,190 @@
+    return arrayNumbers.filter((value) => typeof value === "number").map((num) => num % 3)
+  }
+
+// PASS  ./code-challenges.test.js
+// divisibleByThree
+//   ✓ takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3
+
+// Test Suites: 1 passed, 1 total
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+//describe(name of function as a string, callback function)
+describe("arrayCubed", () => {
+
+   // Given function of an array of numbers with expected outputs
+  const cubeAndSum1 = [2, 3, 4]
+  // Expected output: 99
+  const cubeAndSum2 = [0, 5, 10]
+  // Expected output: 1125  
+
+  //it(description of the function as a string, callback function)
+  it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    
+  //expect(statement with the function call and jest matcher with the expected output) 
+  expect(arrayCubed(cubeAndSum1)).toEqual(99)
+  expect(arrayCubed(cubeAndSum2)).toEqual(1125)
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+// arrayCubed
+//   ✕ takes in an array of numbers and returns the sum of all the numbers cubed (1 ms)
+
+// ● arrayCubed › takes in an array of numbers and returns the sum of all the numbers cubed
+
+//   ReferenceError: arrayCubed is not defined
 
 // b) Create the function that makes the test pass.
+
+// PSEUDOCODE
+// input: two arrays
+// output: two numbers
+
+// input: [2, 3, 4]
+// output: 99
+
+// input: [0, 5, 10]
+// output: 1125
+
+// PROCESS:
+// Create a function named "arrayCubed" that takes in an array of numbers, represented by an argument assigned by the function, set in parentheses named "arrays" and return the sum of all the numbers cubed.
+// Create a return statement to invoke the argument of "arrays" (numbers).
+// Use the '.reduce' built-in method which will reduce the elements in the array and return only the accumulated single number value from the array. 
+// Set the initial value of the function by passing an argument named "sum" (representing the sum of the numbers in the array) and set a value named "number), which represents the number elements in the array that are to be returned.
+// To allow us to write a shorter function, use the arrow function syntax (=>) and call on the sum of the numbers by using the addition operator (+) to produce/connect the sum of the numbers cubed.
+// Pass in an argument that uses the multiplication operator (*) to multiply each number three times, which represents the numbers cubed.
+// Add the index of '0" preceded by a comma (,) and place outside of the math logic to represent the first index number (starting point). 
+// Returns the sum of all the numbers cubed.
```

Comment:
You are definitely on the right track, but misunderstanding `.reduce` a little bit. `.reduce` takes a callback function as an argument. The parameter `sum` represents the "accumulator", while the param `number` represents the current value on each iteration. With each iteration, `sum` is being updated by adding the current value of `sum` to `number * number * number`. The `0` at the end represents what the accumulator (in this case, `sum`) is starting off at.